script.js

```diff
@@ -1,6 +1,27 @@
 /* eslint-disable linebreak-style */
+/* eslint-disable prefer-destructuring */
+/* eslint-disable no-loop-func */
+
 const hamburger = document.querySelector('.hamburger');
 const toolbar = document.querySelector('.mobile-menu');
+const form = document.getElementById('contact');
+const errorMsg = document.getElementById('text');
+const inputName = document.getElementById('name');
+const inputEmail = document.getElementById('email');
+const inputMsg = document.getElementById('msg');
+
+const projectCard = [
+  {
+    name: 'Multi Post Stories',
+    image: 'media/pop-up-mobile.png',
```

Comment:
[ OPTIONAL ] Container becomes blank when the user closes the pop-up on mobile
![mobile](https://user-images.githubusercontent.com/35504149/136652236-aea725c3-9905-42a2-8630-3bfaf5cf46c7.png)
.